### ISSUE TITLE
Invalid hash for YUP

### ISSUE THREAD

**Issue by ryanl-ee** [TDarkShadow/vivanewvegas-wabbajack#14]
I'm getting this issue on a fresh install (fresh Windows, fresh Steam, fresh FNV after following the pre-modlist installation instructions)
```
[ERROR] (Wabbajack.Installer.StandardInstaller) Hashes for mods\Yukichigai Unofficial Patch - YUP\YUP - Base Game + All DLC.esm did not match, expected SLuc29QNfNQ= got EDlird4PXus=
```
I cleared out the VNV install directory and tried again just to be sure.

**Comment by kratores85** [comment]
Yes. We have to wait for VNV WJ modlist update.

**Comment by kratores85** [comment]
This is fixed in v1.5.2
